internal/checker/terraform/rules/azure/azurerm_security_center_contact_phone.rego

```diff
@@ -0,0 +1,63 @@
+# METADATA
+# title: "Confirm that the 'Phone number' is configured for the security contact"
+# description: "Validate that a 'Phone number' is provided and configured for the security contact. This ensures that there is a valid and reachable contact number associated with security-related matters"
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/security_center_subscription_pricing
+# custom:
+#   id: CB_TFAZR_087
+#   severity: LOW
+package lib.terraform.CB_TFAZR_087
+
+import future.keywords.in
+
+supportedResources := [
+	"azurerm_security_center_contact"
+] 
+
+isvalid(block) {
+	block.Type == "resource"
+	some label in block.Labels
+	label in supportedResources
+}
+
+resource[resource] {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource[resource] {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
+pass[resource] {
+	some resource in input
+	isvalid(resource)
+    resource.Attributes.phone != null
+}
+
+pass[resource] {
```

Comment:
Is this block part of the policy? removed if not

Reply:
Thank you this is noted, I will remove all unused code.